packages/kit/src/views/Market/MarketDetailV2/components/SwapPanel/useSwapPanel.ts

```
@@ -0,0 +1,138 @@
+        setInputTokenSymbol(tokenSymbol); // Token to sell
+        if (tokenSymbol === outputTokenSymbol) {
+          const otherToken = MOCK_TOKENS.find((t) => t.value !== tokenSymbol);
+          if (otherToken) setOutputTokenSymbol(otherToken.value);
+        }
+      }
+    },
+    [tradeType, outputTokenSymbol],
+  );
+
+  const totalValue = useMemo(() => {
+    const numericAmount = parseFloat(amount);
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)